experimental/dds/tree2/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -360,12 +362,22 @@ describe("SharedTree benchmarks", () => {
 							schema: wideSchema,
 						});
 
-						const path = wideLeafPath(numberOfNodes - 1);
+						const rootPath = {
+							parent: undefined,
+							parentField: rootFieldKeySymbol,
+							parentIndex: 0,
+						};
+						const nodeIndex = numberOfNodes - 1;
+						const editor = tree.editor.sequenceField({
```

Comment:
Why not use a value field here like you did above? It seems closer to what the original test was trying to do.

Reply:
I did it this way because foo is a list of child nodes (whose values we were setting) for this test while in the other one, foo is a single node with a recursive child under the foo field. I think deleting and inserting nodes in the foo sequence should be roughly equivalent to what the original test was doing. Also, I could definitely be wrong but I don't think there's a way to get a value field from a particular node in a sequence, is there?

Reply:
That makes sense, thanks for explaining.

As to getting the value field from a particular node in a sequence, in case it comes up later, it is possible: you have to pass the correct path to `tree.editor.valueField`:
```
for (let value = 1; value <= setCount; value++) {
	const fieldEditor = tree.editor.valueField({
		parent: {
			parent: rootPath,
			parentField: localFieldKey,
			parentIndex: nodeIndex,
		},
		field: localFieldKey,
	});
	fieldEditor.set(singleJsonCursor(value))
}
```

Reply:
Feel free to resolve this comment thread without further action.

Reply:
Ohhh that's good to know, thanks! I'm not a fan of the API though. Since we're trying to retrieve a node rather than a field, calling `valueField` feels wrong. It's also strange to me that we're specifying `localFieldKey` twice when the node that is under the specified parent doesn't have a `localFieldKey` field.

Reply:
> we're trying to retrieve a node
We are? This the equivalent of `root.foo[nodeIndex].foo = value`, so we're just setting the inner `foo` field here. Maybe I'm not understanding what you mean by "retrieve".

> It's also strange to me that we're specifying localFieldKey twice when the node that is under the specified parent doesn't have a localFieldKey field.
Right, that was just because we'd have to pick some field key to put the value under. It's true that it doesn't exist right now, but we would have needed to add it for this to work. I could have picked "value" instead so that it looked like this:
```
	const fieldEditor = tree.editor.valueField({
		parent: {
			parent: rootPath,
			parentField: localFieldKey,
			parentIndex: nodeIndex,
		},
		field: brand("value"),
	});
```
which would be the equivalent of `root.foo[nodeIndex].value = value`.

Reply:
Ohh I see, I was thinking that it's supposed to be equivalent to just `root.foo[nodeIndex] = value`.